### PR TITLE
Update code to use `Module` attribute

### LINKED ISSUE #9: As a disorganised student, I can store the modules a professor is teaching
... so that I can contact the professors who teach a module which I am currently taking.

### LINKED ISSUE #48: Update code to use the Module attribute
Command includes:
- [x] Add
- [x] Edit

### PR DESCRIPTION
**Pull request by tsulim**
Closes #9, closes #48 
- Add `Module` data field to `Person` model
- Update commands (e.g., `add` and `edit`) to use `Module`
- Add sample `Module` data to `SampleDataUtil`
- Update tests to cover `Module`
- Update test data with sample `Module` data
- Update ui to display `Module` in the application

### CODE REVIEW

File: src/test/java/staffconnect/model/person/ModuleTest.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void isValidModule() {
         assertFalse(Module.isValidModule("cs2103t")); // suffix not capitalised
         assertFalse(Module.isValidModule("cs2103T")); // prefix not capitalised
```
Comment (iynixil) [inline src/test/java/staffconnect/model/person/ModuleTest.java]: Should we make module case-insensitive (but save as uppercase) to allow more flexibility?

**Review by iynixil — COMMENTED** [review]
Looks good! Made a suggestion for Module values.

File: src/test/java/staffconnect/model/person/ModuleTest.java (hunk 1 above)
Comment (whitesnowx) [reply src/test/java/staffconnect/model/person/ModuleTest.java]: Sorry, I think this is under my job. I think I can implement that in the 1.2b, seperate from this PR :statue_of_liberty:

Comment (iynixil) [reply src/test/java/staffconnect/model/person/ModuleTest.java]: Okay, no problem!

**Review by iynixil — APPROVED** [review]

**Pull request merged**